core/controllers/admin_test.py

```diff
@@ -1270,11 +1270,9 @@ def test_grant_super_admin_privileges_fails_with_invalid_username(
     ) -> None:
         self.login(feconf.ADMIN_EMAIL_ADDRESS, is_super_admin=True)
 
-        response = self.put_json(
+        self.put_json(
             '/adminsuperadminhandler', {'username': 'fakeusername'},
-            csrf_token=self.get_new_csrf_token(), expected_status_int=400)
-
-        self.assertEqual(response['error'], 'No such user exists')
+            csrf_token=self.get_new_csrf_token(), expected_status_int=404)
```

Comment:
Hey @LuyaoJ, I think we should check for the actual error messages and not just the status code, this way we will be sure about the message that we will be sending to the frontend. Please make the changes here and elsewhere.
Thanks!

Reply:
@lkbhitesh07 Actually I don't think we need to do that for 404, right? 404 is a generic error message and I don't think we need to keep checking it, just the status int is enough.

Reply:
Ah! okay, Makes sense. Hey @LuyaoJ, you can ignore my comment.
Thanks!